src/attribute.rs

```diff
@@ -105,43 +164,11 @@ impl Attribute {
         unsafe { from_glib_full(pango_sys::pango_attr_underline_new(underline.to_glib())) }
     }
 
-    pub fn new_variant(variant: Variant) -> Option<Attribute> {
+    /*pub fn attr_variant_new(variant: Variant) -> Option<Attribute> {
         unsafe { from_glib_full(pango_sys::pango_attr_variant_new(variant.to_glib())) }
-    }
+    }*/
 
     pub fn new_weight(weight: Weight) -> Option<Attribute> {
         unsafe { from_glib_full(pango_sys::pango_attr_weight_new(weight.to_glib())) }
     }
-
-    pub fn get_attr_class(&self) -> AttrClass {
```

Comment:
Why remove this?

Reply:
Went too quickly I think... Putting it back!